be/src/types/bitmap_value.cpp

```diff
@@ -922,92 +922,138 @@ void BitmapValue::_convert_to_smaller_type() {
     }
 }
 
-int64_t BitmapValue::sub_bitmap_internal(const int64_t& offset, const int64_t& len, BitmapValue* ret_bitmap) {
-    if (offset > 0 && offset >= _bitmap->cardinality()) {
-        return 0;
+void BitmapValue::_convert_to_bitmap() {
+    switch (_type) {
```

Comment:
it was not expected

Reply:
we shouldn't change input bitmap type

Reply:
> we shouldn't change input bitmap type

If not convert to bitmap, maybe it's hard to get sub_bitmap by idx

Reply:
Max size of set is 32, no need to convert to bitmap.